StreamTemp-Integ.py

```diff
@@ -44,7 +44,7 @@
         BATCH_SIZE = b_list
         RHO = 365
         HIDDENSIZE = h_list
-        saveEPOCH = 200   # it was 50
+        saveEPOCH = EPOCH / 5   # it was 50
```

Comment:
If saveEPOCH was 200 and TestEPOCH/EPOCH were 2000, shouldn't this be EPOCH / 10 to be consistent with the manuscript's model runs? Or perhaps better still, set some minimum value? What about
```py
saveEPOCH = max(50, int(EPOCH / 10))
```

Reply:
Wouldn't hurt.  I think the main thing is just that the final epoch is written out without having to hand-edit multiple parameters.